agignore

```diff
@@ -1,3 +1,4 @@
 log
 tags
 tmp
+/.git/
```

Comment:
Do we need the leading slash here? How about `.gitignore` files that may be in subdirectories?